glean-core/src/lib.rs

```diff
@@ -874,6 +874,22 @@ pub(crate) fn register_ping_type(ping: &PingType) {
     }
 }
 
+/// Gets a list of currently registered ping names.
+///
+/// # Returns
+///
+/// The list of ping names that are currently registered.
+pub fn glean_get_registered_ping_names() -> Vec<String> {
+    block_on_dispatcher();
```

Comment:
I think this is the crux why we never implemented it.
This would block if initialize is never called. Or at least block until it is. 

Should we call this out more explicitly? At the very least this needs to be explicitly called out as a debug API?

Reply:
I'll document the behavior. I don't see this being used before Glean is initialized, I really only see the one use case with debugging UI's and tooling so marking this as for debug use seems logicial.